src/GitPrompt.ps1

```diff
@@ -97,6 +97,8 @@ $global:GitPromptSettings = [pscustomobject]@{
 
     EnableWindowTitle                           = 'posh~git ~ '
 
+    AnsiConsole                                 = $Host.UI.SupportsVirtualTerminal -or ($Env:ConEmuANSI -eq "ON")
```

Comment:
You're probably right.  I carry some baggage having written scripts back in the early PS days that were affected by users setting `Set-StrictMode -Version Latest` in their global session.  Fortunately that does not impact modules.  That and I still cling to statically typed notions and referring to a property that may or may not exist just feels wrong.  ;-)